flake.nix

```diff
@@ -78,7 +88,17 @@
           // builtins.mapAttrs (_: crane-lib.cargoDoc) crates
           # fmt
           // builtins.mapAttrs (_: crane-lib.cargoFmt) crates;
-
+        overlays.default = import ./nix/overlays.nix { inherit pkgs packages; };
+        nixosModules.default = import ./nix/genealogos-module.nix;
+        nixosConfigurations.genealogos = nixpkgs.lib.nixosSystem
+          {
+            pkgs = import nixpkgs { inherit system; overlays = [ overlays.default ]; };
```

Comment:
This should ideally not be needed.

Either:
1. Have the NixOS module do it, or (better)
2. Just pass in the `genealogos` package to the module one way or another, and set it as the default value for `services.genealogos.package` (like [here](https://github.com/nix-community/queued-build-hook/blob/6c2f0f695e3bcbe5de28a6a1d66d499c95875ba4/module.nix#L5) for instance)